versions.tf

```diff
@@ -1,4 +1,10 @@
-
 terraform {
-  required_version = ">= 0.12"
+  required_version = ">= 0.12.26"
+
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+      version = ">= 3.22.0, < 4.0.0"
```

Comment:
Any specific reason for restricting the major version here? Common practice is to only require a minimum version, and enforce no other restrictions

Reply:
I saw it on an example for the docs. In that case, I'll remove it!